### DISCUSSION TITLE
Enable captcha by default?

### DISCUSSION THREAD

**Discussion by thisisjamessmith** [verbb/formie#2058] [Q&A] [answered]
I'd like to make it as simple as possible for editors to create forms, and as such I'd like to disable the "Integrations" tab on the forms. But then I also need to apply the captcha integration by default for every form. Is there a way to do that?

**Comment by engram-design** [comment, marked as answer]
The integrations tab does need to stay there, even if you just have a captcha. If you really care about it being shown I would say you could inject some CSS to visually hide it, but Formie still needs it in the form.

As for it being on by default, that's already the case. If you enable a captcha integration, any **new** form is going to have that enabled by default. It's only existing forms where you need to enable the previously disabled captcha integration.

**Reply by thisisjamessmith** [reply]
Ah great thanks I didn't realise new forms would have it enabled by default. In that case surely I'm ok to disable the Integrations tab via the user group settings. (This is a brand new build, so no existing forms yet)
